src/Randomizer.SMZ3.Tracking/AutoTracking/LuaScripts/bizhawk/emulator.lua

```diff
@@ -40,15 +40,22 @@ function emulator.print(message)
 end
 
 function emulator.read_bytes(address, length, domain)
-    return base64_encode(memory.readbyterange(translate_address(address, domain), length, domain), length)
+	if gameinfo.getromname():find("SMZ3 Cas") ~= nil then
```

Comment:
It might be good to document this in the part where we patch the ROM with the name, so we know not to change it